code/datums/holocall.dm

```diff
@@ -159,13 +159,13 @@
 /datum/holocall/proc/Check()
 	for(var/I in dialed_holopads)
 		var/obj/machinery/hologram/holopad/H = I
-		if((H.stat & NOPOWER) || !H.anchored)
+		if(!H.anchored)
 			ConnectionFailure(H)
 
 	if(QDELETED(src))
 		return FALSE
 
-	. = !QDELETED(user) && !user.incapacitated() && !QDELETED(calling_holopad) && !(calling_holopad.stat & NOPOWER) && user.loc == calling_holopad.loc
+	. = (user.loc == calling_holopad.loc) && !user.incapacitated() && !QDELETED(user) && !QDELETED(calling_holopad)
```

Comment:
The order here matters in the way that QDELETED will also check if it's null. Having it in this order might break if say the user is null or the calling_holopad is null

Reply:
Oop yeah, I tried to be clever here but this will runtime if the pad or user are deleted.